HonestHouse/HonestHouse/Sources/Presentation/Main/ViewModel/MainViewModel.swift

```diff
@@ -9,6 +9,41 @@ import SwiftUI
 
 @Observable
 final class MainViewModel {
-    var selectedSegment: MainViewSegmentType = .trishot
+    var selectedSegment: MainViewSegmentType = .trishot {
+        didSet {
+            if selectedSegment != oldValue {
+                exitEditMode()
+            }
+        }
+    }
```

Comment:
계산 프로퍼티로 selectedSegment 값 변화를 감지하는 것 보다 하나의 액션으로 정의해도 좋을 것 같아요
```suggestion
    private(set) var selectedSegment: MainViewSegmentType = .trishot

    func setSelectedSegment(_ segment: MainViewSegmentType) {
        guard selectedSegment != segment else { return }
        selectedSegment = segment
        exitEditMode()
    }
    ...
}
```

Reply:
오 그러네요 반영해보겠습니다 감사합니다!